decidim-accountability/app/packs/src/decidim/accountability/admin/imports.js

```diff
@@ -0,0 +1,16 @@
+const tuggleInfo = (val) => {
+  document.querySelectorAll(".help-text").forEach((toBeHidden) => {
+    if (val === toBeHidden.id && toBeHidden.style.display === "none") {
+      toBeHidden.style.display = null;
+    } else {
+      toBeHidden.style.display = "none";
```

Comment:
Avoid using inline styles when the CSS framework provides us already with utility classes to hide/show elements:
https://get.foundation/sites/docs/visibility.html#generic-hide-classes